p4-14/v1.0.4/tex/p4.pt

```diff
@@ -1977,6 +1976,10 @@ The hash value is used to generate a value between \texttt{base} and
 The result can be used as a hash digest of a long key (such as 5-tuple) or as 
 an offset/index to reference a register array. Normal value conversion 
 takes place when setting \texttt{dest} to the result.
+
+If any of the fields in the \texttt{field_list_calc} have a parent
+header that is not valid, then the operation leaves \texttt{dest} with
+an undefined value.
```

Comment:
I will need to re-read a comment from antoninbas, but this change perhaps needs correction.  I believe he said that if any fields have a parent header that is not valid, bmv2 leaves those fields out of the hash calculation.  Thus the set of fields included in the hash calculation can change from one packet to the next based upon which set of headers are valid for those packets.